net/miniupnpd/Makefile

```
@@ -1,56 +1,52 @@
 PORTNAME=	miniupnpd
 DISTVERSION=	2.3.9
-PORTREVISION=	1
+PORTREVISION=	2
 PORTEPOCH=	1
 CATEGORIES=	net
 MASTER_SITES=	http://miniupnp.free.fr/files/ \
 		https://github.com/miniupnp/miniupnp/releases/download/${PORTNAME}_${DISTVERSION:S|.|_|g}/
 
 PATCH_SITES=	https://github.com/miniupnp/miniupnp/commit/
 PATCHFILES=	7949a7acb3b8471681157863387339ea2ca749fb.patch:-p2
+PATCHFILES=	94322ba9a41c13cc75efa5eff8fada7a77898036.patch:-p2
 
 MAINTAINER=	squat@squat.no
 COMMENT=	Lightweight UPnP IGD & PCP/NAT-PMP daemon which uses pf
-WWW=		http://miniupnp.free.fr/
+WWW=		https://miniupnp.tuxfamily.org/
 
 LICENSE=	BSD3CLAUSE
 
 LIB_DEPENDS=	libpfctl.so:net/libpfctl
 
-USES=		cpe localbase:ldflags ssl
+USES=		cpe
 CPE_VENDOR=	miniupnp_project
 
 USE_RC_SUBR=	${PORTNAME}
 
 HAS_CONFIGURE=	yes
 # unconditionally use pf, ipfw does not work on FreeBSD
-CONFIGURE_ARGS=	--firewall=pf --libpfctl
-
+CONFIGURE_ARGS=	--firewall=pf \
+		--libpfctl --vendorcfg
+MAKE_ENV=	MANPREFIX=${PREFIX}/share
 MAKE_JOBS_UNSAFE=	yes
 
-CFLAGS+=	-I${OPENSSLINC}
-LDFLAGS+=	-L${OPENSSLLIB} -lpfctl
-
 PLIST_FILES=	etc/miniupnpd.conf.sample \
-		share/man/man8/miniupnpd.8.gz \
-		sbin/miniupnpd
+		sbin/miniupnpd \
+		share/man/man8/miniupnpd.8.gz
 
-OPTIONS_DEFINE=			CHECK_PORTINUSE IPV6 LEASEFILE UPNP_IGDV2 \
-				UPNP_STRICT
+OPTIONS_DEFINE=			CHECK_PORTINUSE IPV6 LEASEFILE \
+				UPNP_IGDV2 UPNP_NOPPP UPNP_STRICT
```

I'd try to keep the diff small, also for upstream purposes

```suggestion
OPTIONS_DEFINE=			CHECK_PORTINUSE IPV6 LEASEFILE UPNP_IGDV2 \
			UPNP_NOPPP UPNP_STRICT
```

---

I'm aiming for minimal changes while adding features while trying to remain close to FreeBSD ports state. It's much better to do separate style commits in FreeBSD ports, best when nothing else is changed in order to not cause regressions and make upstreaming/downstreaming easier for everybody.